### ISSUE TITLE
Can cell linkage be used with datasets lacking new expression info?

### ISSUE THREAD

**Issue by hummuscience** [JunyueC/sci-fate_analysis#3]
In figure 3 of the paper you show a 3D plot with timer on the z axis and UMAP1/2 on X/Y.

I understand that the analysis performed was done with the following script: https://github.com/JunyueC/sci-fate_analysis/blob/master/Main_script_cell_linkage_analysis.R

I was wondering if the same analysis/plot can be done using data from a "normal" time course experiment (for example, see Sharma et al 2020 https://www.nature.com/articles/s41586-019-1900-1?platform=oscar&draft=collection).

**Comment by JunyueC** [comment]
Yes - I think the same plot can be shown if you have the information about the cell state transition path at single cell level. This can be inferred by RNA velocity for normal single cell RNA-seq data.

**Comment by hummuscience** [comment]
Will this also work if it is a time course with many days in between instead of a current snapshot of development or a continuous time course of hours (like in your paper)?

**Comment by JunyueC** [comment]
I guess this will be challenging as the newly synthesized mRNA or intronic reads can only be used to infer cell state dynamics across several hours shorter than a whole day
